modals/src/main/java/com/decathlon/vitamin/compose/modals/impl/ModalButtons.kt

```diff
@@ -0,0 +1,78 @@
+package com.decathlon.vitamin.compose.modals.impl
+
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.width
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.unit.Dp
+import com.decathlon.vitamin.compose.modals.ModalButtonsOrientation
+
+internal object ModalButtons {
+    private const val MAX_BUTTONS_COUNT = 3
+
+    @Composable
+    internal fun Buttons(
+        modifier: Modifier,
+        buttons: List<(@Composable () -> Unit)>,
+        buttonsOrientation: ModalButtonsOrientation,
+        topPadding: Dp,
+        buttonsSpacer: Dp
+    ) {
+        if (buttons.isNotEmpty()) {
```

Comment:
We should put this condition outside. Create a composable who can emit nothing is a bad pattern.